lib/SmartyPlugins/modifier.duration_format.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * LMS version 26.x
```

Comment:
A dlaczego tu 26.x?

Reply:
Poprawić na 27.x?

Chciałem zobaczyć jak jest gdzieś indziej i znalazłem brak komentarza w 

https://github.com/chilek/lms-plus/blob/master/userpanel/modules/voip/minibilling.php

Jaka jest polityka z tym?

Reply:
Przeoczenie.

Reply:
Poprawiłem na 27.x

Reply:
A jak jest w innych miejscach? Sugeruję poprawić tak jak jest gdzie indziej.
Albo niepotrzebne spieszysz się gdzieś, albo masz niewłaściwy system decyzyjny :)
Moim zdaniem to ta pierwsza opcja.

Reply:
Mój system decyzyjności nie rozumie idei pisania wersji "1.11-git" kiedy wrzucamy plik do "27.x". 
Mam zawsze opór z robieniem nielogicznych rzeczy a sam wiesz że na pierwszy rzut oka wiele rzeczy się takie wydaje.